README.md

```diff
@@ -1,79 +1,44 @@
 # Microsoft Teams JavaScript client SDK
 
-Welcome to the Teams client SDK monorepo! For breaking changes, please refer to our changelog in the monorepo root of the `2.0-preview` branch.
+Welcome to the Teams client SDK monorepo! For breaking changes, please refer to our changelog in the monorepo root of the `2.0-preview` branch. This repository contains the core teams-client-sdk as well as tools and applications for use analyzing and testing.
```

Comment:
nit: Extra 'use'
This repository contains the core teams-client-sdk as well as tools and applications for analyzing and testing.